website/docs/cli/plugins/index.mdx

```diff
@@ -7,24 +7,24 @@ description: >-
 
 # Managing Plugins
 
-Terraform relies on plugins called "providers" in order to manage various types
-of resources. (For more information about providers, see
+Terraform relies on plugins called providers to manage various types
+of resources. For more information about providers, see
```

Comment:
For context: we use "refer to" to keep our language inclusive for those using screen readers:
```suggestion
of resources. For more information about providers, refer to
```